certified-connectors/AXtension Content Gate/readme.md

```diff
@@ -0,0 +1,32 @@
+## AXtension Content Gate Connector
+AXtension Content Gate helps you find all relevant content within a split second, regardless of storage location. Connect your Microsoft Dynamics 365 environment with AXtension� Content Gate and create a single point-of-truth for all content in your organization.
```

Comment:
Looks like there's an issue with the character here. That should be resolved before merging.

Reply:
Removed the invalid character.